source/Octopus.Tentacle.Tests.Integration/ClientScriptExecutionIsolationMutex.cs

```diff
@@ -22,8 +23,7 @@ public async Task ScriptIsolationMutexFull_EnsuresTwoDifferentScriptsDontRunAtTh
         {
             await using var clientTentacle = await tentacleConfigurationTestCase.CreateBuilder()
                 .WithTentacleServiceDecorator(new TentacleServiceDecoratorBuilder()
-                    .CountCallsToScriptServiceV2(out var scriptServiceV2CallCounts)
-                    .CountCallsToScriptService(out var scriptServiceCallCounts)
+                    .RecordMethodUsage(tentacleConfigurationTestCase.LatestScriptServiceType, out var tracingStats)
```

Comment:
We're not sure why the test case configuration can define the version of the script service to use, but the test still needs to define the version of the command to build. We were getting the impression as we read through the changes that this was heading in the direction that the test would be agnostic of the setup required for different script service / command version combinations. Is that not the case?

Reply:
@david-staniec-octopus The test will always need to use the command from the latest version of the script service. TentacleClient always needs to take the latest version of the `StartScript` command, but internally it may use a different script service version, depending on the capabilities of the client tentacle.

We need the test case to identify what is the latest version of the script service that the tentacle version is running so that we can correctly record the method usage on the correct service (as it'll change depending on the dependent tentacle version).

Happy to clarify on a call if that helps